dotCMS/src/main/java/com/dotmarketing/beans/MultiTree.java

```diff
@@ -107,6 +107,12 @@ private MultiTree(MultiTree tree) {
         this(tree.parent1, tree.parent2, tree.child, tree.relationType, tree.treeOrder, tree.getPersonalization(), tree.getVariantId());
     }
 
+    public static MultiTree vary(final MultiTree multiTree, final String newVariant) {
```

Comment:
javadoc

Reply:
why is it called vary? call it buildMultitreeWithVariant

Reply:
done https://github.com/dotCMS/core/pull/23258/commits/be0dccb64465aa648a685a47be69411aa478f1b7#diff-ee5026536e73ba67dcf187415db7af51b4ea21727f9a73b52ce145bbea53b174R110